experiments/compare_traces.py

```diff
@@ -181,7 +184,7 @@ def load_file(fn: str) -> Tuple[Mission, List[MissionTrace]]:
         with open(fn, 'r') as f:
             jsn = json.load(f)
             mission = Mission.from_dict(jsn['mission'])
```

Comment:
It doesn't really make too much difference here, but you can use `mission.system` instead of `SYSTEM`. I don't mind either way.